ext/cache/sqlite.rs

```diff
@@ -262,10 +270,7 @@ impl Cache for SqliteBackedCache {
           get_responses_dir(cache_storage_dir, request.cache_id)
             .join(response_body_key);
         let file = tokio::fs::File::open(response_path).await?;
```

Comment:
Dang, I thought it might have fixed https://github.com/denoland/deno/issues/19696 😅

Reply:
I could add a quick fix for this but testing is a challenge.